limited/apothecary

```diff
@@ -3,7 +3,7 @@ __Basics__
 Each night, the Apothecary may poison a player, killing them at the end of the next night. Alternatively, they may cure the player they last poisoned.
 __Details__
 The Apothecary starts the game with three flasks of poison and a single antidote.
-When the Apothecary poisons a player, that player will be killed at the end of the next night (this is not an attack). This will only happen if the Apothecary survives until the end of that night. 
+When the Apothecary poisons a player, that player will be killed at the end of the next night (this is not an attack). The Apothecary will only poison a player if the Apothecary survives the night. 
```

Comment:
Then why is it even an immediate ability? No this doesnt make any sense. If anything the poison should kill no matter what. It's immediately applied and then death is sure (unless cured or protected). This would be a further buff but I dont see the problem as this doesnt make it stronger than roles as Assassin still